iterm2_dwim/__init__.py

```diff
@@ -5,12 +5,13 @@
 import sys
 import traceback
 
-from iterm2_dwim.editors import emacs
+from iterm2_dwim.editors import emacs, sublime
+from iterm2_dwim.settings import editor
 from iterm2_dwim.logger import log
 from iterm2_dwim.parsers import get_path_and_line
 
 
-Editor = emacs.Editor
+Editor = sublime.Sublime if editor.lower() == 'sublime' else emacs.Emacs
```

Comment:
If you approve of my suggestion below, here would be slightly different, something like

```python
if settings.editor.sublime_path:
    Editor = sublime.Sublime(settings.sublime_path)
elif settings.editor.emacsclient_path:
    Editor = emacs.Emacs(settings.emacsclient_path)
else:
    exc = Exception('No editor specified in settings.py')
    notify(exc)
    raise exc
```